### DISCUSSION TITLE
display multiple camera streams when reviewing stored video

### DISCUSSION THREAD

**Discussion by capnjip** [blakeblackshear/frigate#11537] [Ideas]
Hello, In other security camera software platforms it is often possible to select several cameras to simultaneously play while the user scrubs through a single timeline of the stored/recorded streams. This makes tracking objects around the monitored property easier. Is this feature possible in Frigate? If not, I would greatly appreciate the addition. Thank you!

**Comment by hawkeye217** [comment]
This feature is implemented in the next version of Frigate (0.14), which just released beta 1 yesterday.

**Reply by jchretien** [reply]
I briefly tried 14-rc1 and I don't think there was a timeline in the Group Tab.

What I imagine is a single Tab with cameras. From there user can position each camera in the order he wants. Have a single horizontal timeline scrollbar to see every camera image at the chosen time, if one camera didnt record anything at the chosen time it would temporarily shows last frame recorded.

**Reply by NickM-27** [reply]
It does, on the motion tab. Every camera is listed there and you have one timeline to scrub.

**Reply by jchretien** [reply]
Following your answer I just re-updated to 14-rc, and I found out cool stuff I haven't noticed earlier.

I don't find the motion tab, is it an optional tab we can enable somehow?

Cheers

**Reply by NickM-27** [reply]
![Screen Shot 2024-08-01 at 07 55 06 AM](https://github.com/user-attachments/assets/90864805-60ec-4aa5-8549-945d19ae6f64)

**Reply by angus2064** [reply]
Hello ! Have you planned to improve this tabs in future versions ? This would be great if we could look at the records of all cams simultaneously and in high res at 1x speed, like in blue iris or surveillance station for exemple. This is actually very difficult to find and especially track what I looking for in motion tabs through my 9 cams because of low res and 4x speed.

**Reply by hawkeye217** [reply]
This is already possible in the History view in Frigate 0.14 and later.

**Reply by angus2064** [reply]
Is this also possible on iPad and iPhone (web or app) ? Because I not succeed to show more than one camera at a time in the history view actually.

**Reply by NickM-27** [reply]
iPhones are not capable of decoding that much activity and don't have the screen real estate to show this. iPads already support this as well
